pages/linux/gs.md

```diff
@@ -0,0 +1,27 @@
+# gs
+
+> GhostScript is a PDF and PostScript interpreter.
+
+- To view a file:
+
+`gs -dQUIET -dBATCH {{file.pdf}}`
+
+- Reduce PDF file size to 150 dpi images for reading on a ebook device:
+
+`gs -dNOPAUSE -dQUIET -dBATCH -sDEVICE=pdfwrite -dPDFSETTINGS=/ebook -sOutputFile={{output.pdf}} {{input.pdf}}`
+
+- Convert PDF file to an image:
+
+`gs -dQUIET -dBATCH -dNOPAUSE -sDEVICE=jpeg -r150 -dTextAlphaBits=4 -dGraphicsAlphaBits=4 -dFirstPage=1 -dLastPage=3 -sOutputFile={{output_%d.jpg}} {{input.pdf}}`
+
+- Extract pages from a PDF file:
+
+`gs -dQUIET -dBATCH -dNOPAUSE -sDEVICE=pdfwrite -dFirstPage=1 -dLastPage=3 -sOutputFile={{output.pdf}} {{input.pdf}}`
```

Comment:
1 and 3 should be in tokens. Also mention that in the description.

Reply:
done